### ISSUE TITLE
Review installation instructions

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop-ops/backdropcms.org#433]
Everytime I have a look at the installation instructions on https://backdropcms.org/installation I'm confused about order and description of the installation steps:

The first sentence of section `3. Update Backdrop's settings.php file` is:

> Skip to the next step and see if you can run the installer.

The "next step" (`4. Set the permissions/ownership on the files directory`) doesn't refer to the installer.
The same applies for the step after the next (`5. Replace the user:pass segment of the connection string with the database username and password` ...), even worse: this one is again about updating `settings.php`.
In contrast, the installer is only and finally mentioned in step 6 (`Run the installer`).

I suggest to rephrase the first sentence of section 3, something like this:

> Skip to step 6 and see if you can run the installer.

> Skip to the last step and see if you can run the installer.

In my opinion it makes even more sense to move the installer section up, leading to more straightforward instructions in the following order:

1. Download Backdrop and extract the files
2. Create the MySQL database
3. Run the installer
4. Troubleshooting / Additional steps
(a) Update Backdrop's settings.php file (current steps 3 and 5)
(b) Set the permissions/ownership on the files directory (current step 4)

**Comment by Graham-72** [comment]
@olafgrabienski I agree with your suggestion to move the installer section up to be step 3. The present instructions make the procedure sound over-complicated. On my server I find the installer does a grand job of updating the settings.php file and setting up the files directory, so it would be good to move these instructions to a section 4 as you propose.

**Comment by jenlampton** [comment]
How's this? https://backdropcms.org/installation

**Comment by olafgrabienski** [comment]
@jenlampton Yes, the instructions look much better with the new ordered sections, so the main issue is fixed in my opinion!

I find some sections still quite verbose, but I have no time to suggest more straightforward sounding alternatives at the moment.

Apart from that, I found two typos:

> ~~provide~~**Provide** some basic web site settings like the Site name, Logo, and email address (the website will use this ~~emial~~ **email** address [...])

**Comment by jenlampton** [comment]
I fixed the caps issue, and removed the email section since we removed that from the installer: https://backdropcms.org/installation.  I also took a stab at decreasing some verbosity, but we could probably still make this more concise.
